deploy/assisted-service.yaml

```diff
@@ -26,6 +26,16 @@ spec:
           imagePullPolicy: Always
           ports:
             - containerPort: 8090
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 8090
+            initialDelaySeconds: 30
```

Comment:
Why do we need to wait 30 seconds? 
I can understand for the first deployment, because it will need to download iso generation (dummy-iso) but if you just reboot or change some env it will take less a second to start.

Reply:
I don't mind removing it but that means that on the first run 2 dummy-image jobs will be created, because k8s will restart the pod by thinking it is not alive.
@filanov

Reply:
You can see the resolved thread above

Reply:
Then 30 seconds is not enough as well. 
Some times it takes minutes until we mange to download the image

Reply:
We needed it mostly for the liveness so k8s don't restart the pod by thinking it is not healthy only because the create-dummy-image job hasn't finished yet and the web server hasn't been initialised yet.

We wanted to use https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes but this feature doesn't exist in OCP < 4.5 so we ended with adding another endpoint that will be reachable before all other endpoints just so that k8s can know the pod is health without waiting 10 minutes each time (because this is the worst case)
@eranco74